multicloud-bootstrap/init.sh

```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+#
+# This is the init script that will call the individual Cloud specific script
+#
+
+## Inputs
+export CLOUD_TYPE=$1
```

Comment:
as per @santoshjpawar this env var should be changed to 
CLUSTER_TYPE because roles are consuming same value as CLUSTER_TYPE
[Discussion thread](https://ibm-watson-iot.slack.com/archives/C031QACEAT1/p1647240145105549?thread_ts=1646900393.727049&cid=C031QACEAT1)

Reply:
Done, I have made the changes